libraries/SrcWrapper/src/stm32/stm32_eeprom.c

```diff
@@ -151,8 +151,17 @@ static uint8_t eeprom_buffer[E2END + 1] __attribute__((aligned(8))) = {0};
   */
 uint8_t eeprom_read_byte(const uint32_t pos)
 {
+#if defined(DATA_EEPROM_BASE)
```

Comment:
Here there will be a misalignment between the buffered eeprom value an the real value.

Reply:
when writing the EEPROM, align eeprom_buffer[pos] = (uint8_t)data;